### DISCUSSION TITLE
show languages in component only if user has permission

### DISCUSSION THREAD

**Discussion by alexeysvyridov** [WeblateOrg/weblate#5110] [Q&A] [answered]
I have component which has many translations.
I created a group and set permission for the user, which can translate only "French", i want that user have able to see, and translate only this language in this component. How solve this?

**Comment by alexeysvyridov** [comment]
![Capture](https://user-images.githubusercontent.com/33163048/103226065-f0614b80-4933-11eb-9177-81052194e523.PNG)

**Comment by nijel** [comment, marked as answer]
Right now there is no way to limit visibility of other languages, only other permissions for them. In many cases, it's useful for a translator to be able to see other languages (for example to better understand how others are translating the string).

**Reply by ianathompson** [reply]
Second this. It is very helpful for our Italian Transcreator to have access to the Spanish strings that have been localized already. These can also start discussions between Transcreators as to which translation might be the best one. Suggestion might be adding the localization of French to the WATCH list of the user and having them change their dashboard settings to match.
